opm/autodiff/SimulatorBase_impl.hpp

```diff
@@ -180,7 +180,10 @@ namespace Opm
 
             // write the inital state at the report stage
             if (timer.initialStep()) {
-                output_writer_.writeTimeStep( timer, state, well_state );
+                // No per cell data is written for initial step, but will be
```

Comment:
Although the comment should be amended a little first: It should not be "no per cell data" but "no per cell property data" I think (per cell data for pressures, saturations etc. are written).